doc/radosgw/dynamicresharding.rst

```diff
@@ -185,27 +190,28 @@ Troubleshooting
 
 Clusters prior to Luminous 12.2.11 and Mimic 13.2.5 left behind stale bucket
 instance entries, which were not automatically cleaned up. This issue also affected
-LifeCycle policies, which were no longer applied to resharded buckets. Both of
+lifecycle policies, which were no longer applied to resharded buckets. Both of
 these issues could be worked around by running ``radosgw-admin`` commands.
 
-Stale instance management
+Stale Instance Management
 -------------------------
 
-List the stale instances in a cluster that are ready to be cleaned up.
+List the stale instances in a cluster that are ready to be cleaned up:
+
+.. prompt:: bash #
 
-::
+   radosgw-admin reshard stale-instances list
 
-   # radosgw-admin reshard stale-instances list
+Clean up the stale instances in a cluster:
 
-Clean up the stale instances in a cluster. Note: cleanup of these
-instances should only be done on a single-site cluster.
+.. prompt:: bash #
 
-::
+   radosgw-admin reshard stale-instances delete
 
-   # radosgw-admin reshard stale-instances delete
+.. note:: Cleanup of stale instances should not be done in a multisite environment.
```

Comment:
I think s/environment/deployment/

Reply:
This language came from https://ceph--63163.org.readthedocs.build/en/63163/radosgw/dynamicresharding/#multisite so I'll update that also.